Testscripts/Linux/ETHTOOL-CHECK-FEATURES.sh

```diff
@@ -37,6 +37,12 @@ if ! VerifyIsEthtool; then
     exit 0
 fi
 
+if [[ $DISTRO == "ubuntu_12.04"* ]]; then
+    LogErr "This distro $DISTRO does not support $ETHTOOL_KEYS features"
```

Comment:
better to use tool to check, not filter using distro

Reply:
I agree but this has been verified with @haiyangz Also like this case, tools might not be available per distro repo condition. I'd rather rely on the OS version. If this is like legacy OS, it would be ok.

Reply:
if the Linux next kernel built on the distro, actual it supports but filter out by this condition.

Reply:
Sent the question to Haiyang. As Joe mentioned there is a chance to have a new kernel in the legacy kernel only if security fix.